modules/monitoring-setting-up-metrics-collection.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="setting-up-metrics-collection_{context}"]
+= Setting up metrics collection
+
+To use the metrics exposed by your service, you need to configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a ServiceMonitor, a custom resource definition (CRD) that specifies how a service should be monitored, or a PodMonitor, a CRD that specifies how a pod should be monitored. The former requires a Service object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a Pod.
+
+This procedure shows how to create a ServiceMonitor for the service.
+
+.Prerequisites
+
+* Log in as a cluster administrator or a user with the `monitor-crd-edit` role.
+
+.Procedure
+
+. Create a YAML file for the ServiceMonitor configuration. In this example, the file is called `example-app-service-monitor.yaml`.
+
+. Fill the file with the configuration for creating the ServiceMonitor:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
nit: we can also at least mention `PodMonitors` or even give an example if no service is exposed.

Reply:
What other mention of `PodMonitor`s do you suggest apart from the brief mention in the introduction? Regarding an actual example, I think @sichvoge recons that we should go with only a `ServiceMonitor` example.

Reply:
Just a nit, if @sichvoge wants emphasis on ServiceMonitor only, I am fine.

Reply:
Yep, @sichvoge confirmed to me that this scope is good. Thanks for the feedback, @s-urbaniak .